### PR TITLE
Bump to Kubernetes 1.21 libraries

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Review by adiprerepa — APPROVED** [review]

**Comment by howardjohn** [comment]
blocked by https://github.com/istio/istio/pull/32131

**Review by liamawhite — APPROVED** [review]

**Review by linsun — APPROVED** [review]
approved pending tests pass

**Comment by howardjohn** [comment]
opened https://github.com/kubernetes/kubernetes/issues/101384

**Pull request merged**
